edx_name_affirmation/models.py

```diff
@@ -45,6 +50,19 @@ class Meta:
         db_table = 'nameaffirmation_verifiedname'
         verbose_name = 'verified name'
 
+    @property
+    def verification_attempt_status(self):
+        "Returns the status associated with its SoftwareSecurePhotoVerification with verification_attempt_id if any."
+
+        if not self.verification_attempt_id or not SoftwareSecurePhotoVerification:
+            return None
+
+        # breakpoint()
+
+        verification = SoftwareSecurePhotoVerification.objects.get(id=self.verification_attempt_id)
```

Comment:
It's unlikely, but this may raise a `SoftwareSecurePhotoVerification.NotFound` error if there isn't a record in the database with this ID.

Could you handle the potential for this exception, please?